### DISCUSSION TITLE
Consider bring VLM into RagFlow project?

### DISCUSSION THREAD

**Discussion by simoncai519** [infiniflow/ragflow#1725] [Ideas]
In generation, VLM can answer questions based on picture stored in knowledge base, instead of parsed text from image. Has this been considered?

**Comment by yingfeng** [comment]
Hi, actually VLM has already been integrated already. There are a bag of multi modal models including openai, openrouter, tongyi-qianwen, gemini, stepfun, and you can decide which one to use. These models will convert the images into text.
